plugins/api-docs/README.md

```diff
@@ -160,6 +160,18 @@ by this plugin.
 Grab a copy of [oauth2-redirect.html](https://github.com/swagger-api/swagger-ui/blob/master/dist/oauth2-redirect.html)
 and put it in the `app/public/` directory in order to enable Swagger UI to complete this redirection.
 
+This also may require you to adjust `Content Security Policy` header settings of your backstage application. So javascript on `oauth2-redirect.html` can be executed.
+
+There are two steps:
+1. Open `oauth2-redirect.html` for editing and add `nonce` to the `script` tag. Like this
+  ```
+  <script nonce="oauth2-redirect">
+  ```
+2. Now adjust backstage configuration, `backend.csp` section and add there new property:
+  ```
+  script-src: ["'self'", "'nonce-oauth2-redirect'", "'unsafe-eval'"]
```

Comment:
This method of using the nonce is almost the same as using `unsafe-inline`, so tbh I would rather suggest that instead, to avoid providing a false sense of security.

Another option would be if we could apply a separate CSP policy for just the `oauth2-redirect.html` page, which would be more secure?

Reply:
this change changes `README.md` and does not change any defaults. So security stays in hands of the person who does that. That's why I thought that doing this as `nonce`, we will make "what's going on" more explicit.

Reply:
Right, but it's an arguably insecure pattern so I think it's best not to recommend it in the readme : /

Reply:
I'm fine with that, but this thing needs to be mentioned in `README.md` somehow, because it's required in any form and without it it will not work. I've spent about 2h of my time to figure this out and want to save this time for others. I'm also fine with any other solution, but I strongly believe this shall be a part of the documentation.

Reply:
Is there any way that we could calculate the hash of the script? Even if manually (adding instructions in the docs how to do it if you modify that script) - and then to supply that hash instead of a nonce. That would be the secure way to do it, if convenient enough to perform.

https://content-security-policy.com/hash/

Reply:
Well the script is static, let's just calculate the hash and document it :grin: 👌